mlrun/common/schemas/model_monitoring/model_endpoints.py

```diff
@@ -104,6 +104,7 @@ def from_flat_dict(cls, endpoint_dict: dict, json_parse_values: list = None):
         )
 
     @validator("monitor_configuration")
+    @classmethod
     def set_name(cls, monitor_configuration):
```

Comment:
Again on another dependency.
The official documentation of the legacy version we use contains this mistake:
https://docs.pydantic.dev/1.10/usage/validators/

I assume it works even in version 1.10 as I see it in other places.